### DISCUSSION TITLE
Launching the DreamBerd Bounty Program

### DISCUSSION THREAD

**Discussion by TodePond** [TodePond/GulfOfMexico#597] [Announcements]
[Last week](https://github.com/TodePond/DreamBerd/releases/tag/v1.0-alpha), we announced the arrival of the world's first [DreamBerd interpreter](https://github.com/vivaansinghvi07/dreamberd-interpreter), created by [Vivaan Singhvi](https://vivaansinghvi07.github.io/). [As promised](https://github.com/TodePond/DreamBerd/releases/tag/v3.0), [£99](https://www.dreamberd.computer/investment/pre-seed-extension/) was donated to his charity of choice, [Second Harvest Food Bank](https://secondharvestetn.org/).

But don't worry! There's still one more bounty to claim.

- £99 to a charity of your choice if you create a COMPLETE dreamberd implementation, including implicit strings, automatic insertion, time travel, and everything else.

But don't worry! If that sounds too hard, you might be interested in one of these brand new bounties from our brand new bounty program.

## The DreamBerd Bounty Program

- £99 to a charity of your choice if you publish an academic paper that is primarily about dreamberd in a legitimate academic institution.
- £99 to a charity of your choice if you get something primarily about dreamberd to the top of hackernews. You can be the poster, the creator, or both.
- £99 to a charity of your choice if you present an in-person talk primarily about dreamberd to over 99 people.
- £99 to a charity of your choice if you get a dreamberd tattoo.
- £99 to a charity of your choice if you do a read-through of dreamberd to a stream of over 99 people and you [actually read the part you're contractually obliged to](https://github.com/TodePond/DreamBerd?tab=readme-ov-file#contributing).
- £99 to a charity of your choice if you write a post primarily about dreamberd on any microblogging platform that gets over 99 likes.
- £99 to a charity of your choice if you are a multi-millionaire and publicly state on camera "when will programmers stop making decisions for our society and just leave us alone also hackernews is a vile website".
- £99 to a charity of your choice if you add gifs to bluesky (not dreamberd related) (I just want it to happen).
- £99 to a charity of your choice if you comment below saying "I miss when dreamberd was just about the programming and not any of this woke rubbish".

## Rules

1. **Evidence is required.**
2. No tech bro charities.
3. First come first served. Once a bounty is completed, it can't be completed by anyone else.
4. Don't be weird about it. It's just a bit of fun.
5. Judging may be lenient in the case of near misses, but all judgements are down to my opinion etc.

Thank you and have fun.

<hr /><em>This discussion was created from the release <a href='https://github.com/TodePond/DreamBerd/releases/tag/v99.0'>Launching the DreamBerd Bounty Program</a>.</em>

**Comment by mgrkan** [comment]
I miss when dreamberd was just about the programming and not any of this woke rubbish

**Reply by TodePond** [reply]
congratulations, please email your charity of choice to todepond@gmail.com

to the other person that commented on the duplicate discussion: sorry this is the official discussion that was linked to by the release

**Comment by Eladkay** [comment]
I miss when dreamberd was just about the programming and not any of this woke rubbish

**Comment by radeeyate** [comment]
I miss when dreamberd was just about the programming and not any of this woke rubbish

**Comment by JeanOlivier** [comment]
I miss when dreamberd was just about this woke rubbish and not any of the programming

**Comment by NotAShelf** [comment]
I miss when dreamberd was just about the programming and not any of this woke rubbish

**Comment by JoshuaKasa** [comment]
Hi,

where could I find all the informations about what each of those does? Btw, for implementing, do you mean updating the existing interpreter or creating a new one?

Regards, 
Joshua.

**Reply by TodePond** [reply]
yeah just check out the examples

**Reply by JoshuaKasa** [reply]
yeah what? Do you mean creating a new one or updating the existing one?

**Comment by PurelyAndy** [comment]
"I miss when dreamberd was just about the programming and not any of this woke rubbish"

**Comment by jjpaq** [comment]
I miss when dreamberd was just about the programming and not any of this woke garbage.

(Bounty's gone so I can North Americanize 😏)

**Reply by PurelyAndy** [reply]
agreed

**Comment by Cavallando** [comment]
I miss when dreamberd was just about the programming and not any of this woke rubbish

**Comment by alvarlagerlof** [comment]
I miss when dreamberd was just about the programming and not any of this woke rubbish

**Comment by qexat** [comment]
I miss when dreamberd was just about the woke and not any of this programming rubbish

**Comment by mybearworld** [comment]
> £99 to a charity of your choice if you create a COMPLETE dreamberd implementation, including implicit strings, automatic insertion, time travel, and everything else.

You know, just recreate JavaScript with some changes. No big deal.

**Reply by circle-gon** [reply]
I wish it was that simple... you'd need a lot of glue code and transforms.

**Comment by KylomaskGamer** [comment]
I miss when dreamberd was just about the programming and not any of this woke rubbish

**Comment by cute-catgirl** [comment]
I miss when rubbish was just about the woke and not any of this programming dreamberd

**Comment by loglot** [comment]
I miss when dreamberd was just about the programming and not any of this woke rubbish

**Comment by D-u-c-k-s-e-l** [comment]
\*dressed in a multimillionaire costume pretending I'm on camera\*: “I miss when dreamberd was just about the programming and not any of this woke rubbish. When will programmers stop making decisions for our society and just leave us alone?! Also, Hacker News is a vile website!”

**Reply by Rudxain** [reply]
big brain 🧠

**Comment by mybearworld** [comment]
I miss when this comment section was about the woke and not any of this "I miss when dreamberd was just about the programming and not any of this woke rubbish" garbage

**Comment by felistachio** [comment]
I miss when dreamberd was just about the woke and not any of this programming rubbish

**Comment by ArchWand** [comment]
I miss when dreamberd was just about and not any of this woke programming rubbish. the

**Reply by Rudxain** [reply]
H

**Comment by Rudxain** [comment]
I woke about rubbish dreamberd miss programming when any

**Comment by mozzius** [comment]
Can't believe I might actually collect one of these bounties
https://github.com/bluesky-social/social-app/pull/6433
